deploy/nl/embeddings.yaml

```diff
@@ -27,6 +27,10 @@ indexes:
     store: MEMORY
     embeddings: gs://datcom-nl-models/embeddings_undata_2024_03_20_11_01_12.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
     model: ft-final-v20230717230459-all-MiniLM-L6-v2
+  undata_ilo_ft:
+    store: MEMORY
+    embeddings: gs://datcom-nl-models/embeddings_undata_ilo_2024_05_15_11_18_05.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
+    model: ft-final-v20230717230459-all-MiniLM-L6-v2
```

Comment:
nit: when you sync, you might also need to add this to https://github.com/datacommonsorg/website/blob/3a2854f075b4614baea488410469ba2feffd16e2/deploy/helm_charts/envs/unsdg.yaml#L45

Reply:
Added!